### PR TITLE
docs: update blog and delete events

### PR DESCRIPTION
**Pull request by yzeng25**
Changes:

- deleted 'Events' tab under 'Resources' tab
Snipaste_2021-09-23_16-28-51

- migrated documents under 'Events' tab to 'Blog' tab, and added tags such as 'Events', and 'Release'.
Snipaste_2021-09-23_16-30-51

- deleted release documents under 'Events' tab, since they are duplications of those under 'Release' tab, and materials under 'Release' tab can be generated automatically.
Snipaste_2021-09-23_16-57-47

- updated tags to Start with capitalized letters,  i.e. 'news to News', 'practical case' to 'Practical Case'
Snipaste_2021-09-23_16-27-40

- changed the link on home page (https://apisix.apache.org/)  to blog with 'Events' tag 
Snipaste_2021-09-23_16-29-30

- updated events.json `blog/tags/Events` to link events to blogs
- modified index.js to update prefix to `/blog` 
- deleted service mesh docs because they belong to api7

### CODE REVIEW

File: website/static/data/events.json [hunk 1]
```diff
@@ -1,12 +1,30 @@
 [
+  {"title": "Apache APISIX Weekly Report",
```
Comment (juzhiyuan) [inline website/static/data/events.json]: Please format this file

**Review by juzhiyuan — APPROVED** [review]
May I know how to distinguish contents in English and Chinese?

File: website/static/data/events.json (hunk 1 above)
Comment (yzeng25) [reply website/static/data/events.json]: fixed

**Comment by yzeng25** [comment]
> May I know how to distinguish contents in English and Chinese?

Yes. This pull request does not solve the problem of separating blogs in English and Chinese. It is an enhancement and a fix on current blogs and events.

For reference, https://github.com/apache/apisix-website/pull/626 could solve the problem of separating blogs in English and Chinese. It creates subdirectories `./i18n/zh` and `./i18n/en` under `./website`, and sorts blogs into these subdirectories in order to distinguish contents in English and Chinese.

**Comment by juzhiyuan** [comment]
cc @Baoyuantop @guoqqqi to have a look when you have time :)

**Review by iamayushdas — APPROVED** [review]
LGTM - EN docs and docusaurus.config.js files 
Thanks :)

**Review by Baoyuantop — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Comment by guoqqqi** [comment]
LGTM, very thanks～

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**